app/stores/ada/TrezorConnectStore.js

```diff
@@ -179,7 +178,7 @@ export default class TrezorConnectStore extends Store implements HWConnectStoreT
 
       // TODO: [TREZOR] fix type if possible
       const trezorResp = await TrezorConnect.cardanoGetPublicKey({
-        path: derivePathPrefix()
```

Comment:
to make the life easier for reviewers, we should avoid making other type of changes in this PR that is focused in design fixs.

Reply:
This needed to be done so that they could test Trezor UI while waiting for my fixed to be merged on to develop. This change can be removed now since develop has the proper fix for Trezor @sarveshwar-singh @YankoFelipe